appserver/persistence/pom.xml

```diff
@@ -62,5 +62,6 @@
         <module>entitybean-container</module>
         <module>jpa-container-l10n</module>
         <module>cmp-l10n</module>
+        <module>data-core</module>
```

Comment:
I wouldn't necessarily have this under the `persistence` module.
I think this should simply sit above under appserver.

I'm imagining a structure somewhat akin to:

* appserver
  * data
    * data-core
    * data-persistence
    * data-nosql

WDYT?